versions/gui_gui.py

```diff
@@ -0,0 +1,101 @@
+import pyperclip
+import os
+import sys
+from tkinter import *
+from tkinter import ttk
+from functools import partial
+from PIL import Image, ImageTk
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from library.TextConverter import TextConverter
+converter = TextConverter()
+text_label_column=0
+copy_button_column=1
+text_output_column=2
+dir = os.path.dirname(__file__)
+
+def copy_to_clipboard(text):
+    pyperclip.copy(text['text'])
+
+def add_text(*args):
+    new_text = translate_text.get()
+    x_size=int(8.5*len(new_text)+240)
+    root.geometry(f"{x_size}x{350}")
+    reversed_text['text'] = converter.reverse_text(new_text)
```

Comment:
these could all be done using dictionary probably but I'm not sure about capitalized and lowercase text

Reply:
I currently do not know how to use a Dictionary in python

Reply:
learn